docs/javascript/v2/how-to-guides/enable-content-sharing/pdf-annotations.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Share and Annotate PDFs
+nav: 1.35
+---
+
+You can now enable PDF annotations and sharing, allowing users to share and annotate PDFs in real-time, providing a rich immersive experience without tab switching.
```

Comment:
also good to call out it's availalbe in chrome only. and maybe a reason on why that is the case?

Reply:
Not chrome only, works on other browsers - only region capture isn't available on other browsers so the viewer experience is = a tab being screen shared